src/telalogin/token.ts

```diff
@@ -0,0 +1,13 @@
+import jwt = require('jsonwebtoken');
+
+export function setToken (email: string): string {
+    const header = {
+        "alg": "HS256",
+        "typ": "JWT"
+    }
+    const payload = {
+        "name": email
+    }
+    const secret = 'umafrasequalquerparateste';
```

Comment:
Como esse `secret` será o mesmo para qualquer token gerado no seu server, você pode defini-lo fora da função. Assim você não fica criando essa variável sempre que um token for gerado. Concorda?